force-app/main/default/classes/DisplayMapController.cls

```diff
@@ -0,0 +1,11 @@
+public with sharing class DisplayMapController {
+    @AuraEnabled (cacheable=true)
+    public static List<Public_Event__c> getEventLocations(String eventIdInitial){
+        
+        //for home page
+        //String str = eventNameInitial + '%';
```

Comment:
@t-gul please remove unnecessary code lines that are commented out and add some comments and descriptions to your code.

Use the below for class description. 
/**
 * @author
 * @date
 * @description - <explain the purpose and where it is being used or from where it is being called>
*/
Use below for the method description.
 /**
    * @param <add the name of the parameter, data type, and purpose>
    * @description <explain what's happening in the method>
    * @returns <what's being returned and where it will be used>
    */ 

@bmtemakina  FYI